data/reserved_words_mysql.txt

```diff
@@ -705,6 +712,7 @@ unsigned
 until
 update
 upgrade
+url
```

Comment:
Not present in [MySQL 8.4 New Keywords and Reserved Words](https://dev.mysql.com/doc/refman/8.4/en/keywords.html#keywords-new-in-current-series),
but added in 8.0.32